transcoder/output/json/JsonOutputManager.py

```diff
@@ -0,0 +1,104 @@
+#
+# Copyright 2022 Google LLC
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import json
+import os
+import sys
+
+#from genson import SchemaBuilder
+from transcoder.message import DatacastSchema, DatacastField
+from transcoder.output import OutputManager
+
+class JsonOutputManager(OutputManager):
+    """Transcode messages to JSON encoding persisted to a file"""
+    def __init__(self, prefix: str, output_path: str, lazy_create_resources: bool = False):
+        super().__init__(lazy_create_resources=lazy_create_resources)
+        self.prefix = prefix
+        self.schemas = {}
+        self.writers = {}
+
+        if output_path is None:
```

Comment:
Not to be resolved now, but in pylint this block keeps coming up as a duplicate and I've been ignoring it. We can probably take this chunk and move to a static function to be used across the various output managers that have this similar functionality.

Reply:
agreed